### DISCUSSION TITLE
Two custom queries return the same columns

### DISCUSSION THREAD

**Discussion by kevinburke1** [sqlc-dev/sqlc#1183] [Q&A]
If I have two queries that each select the same subset of columns on the same table, sqlc creates two different structs instead of one. Is there a way to get them to return the same struct?

We could potentially do this with type aliases in the same package.

**Comment by kyleconroy** [comment]
There is sadly not a way to do this right now. The main issue is that we don't have a syntax for declaring the return type of a query. Thoughts on what that should look like?
